app/templates/gulp/_gettext.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+var gulp = require('gulp');
+
+var $ = require('gulp-load-plugins')();
+
+var paths = gulp.paths;
+
+gulp.task('gettext:extract', function () {
+  return gulp.src([
+    paths.src + '/*.html', paths.src + '/{app,components}/**/*.html',
+    paths.tmp + '/serve/*.html', paths.tmp + '/serve/{app,components}/**/*.html'
+  ], {base: '.'})
+    .pipe($.angularGettext.extract('template.pot'))
+    .pipe(gulp.dest(paths.src + '/app/gettext'));
+});
+
+gulp.task('gettext:compile', function () {
+  return gulp.src(paths.src + '/app/gettext/**/*.po')
+    .pipe($.angularGettext.compile({format: 'json'}))
+    .pipe($.extend('gettext.catalog.json')) // use .json extension for wrap plugin to load content
+    .pipe($.wrap(
+      '\'use strict\';\n\n' +
+      '/**\n' +
+      ' * This module is generated from PO language files located in "' +  paths.src + '/app/gettext" directory.\n' +
+      ' * Use a PO file editor like POEdit (http://poedit.net/) to add translations in these files.\n' +
+      ' * For more information, angular-gettext documentation is available at https://angular-gettext.rocketeer.be/\n' +
+      ' */\n' +
+      'angular.module(\'<%= appName %>\').run([\'gettextCatalog\', function (gettextCatalog) {\n' +
+      '/* jshint -W100,-W109 */\n' +
+      '<'+'% _.forEach(contents, function(translations, lang) { %'+'>'+
```

Comment:
The `'<' + '%'` tricks are to avoid generator template engine to interpret those lines. It must be interpretted by `gulp-wrap` plugin after generation.